spec/features/projects_spec.rb

```diff
@@ -52,6 +52,7 @@
     it { page.should have_content 'Builds chart for last week' }
     it { page.should have_content 'Builds chart for last month' }
     it { page.should have_content 'Builds chart for last year' }
-    it { page.should have_content 'Commit duration in minutes for last 30 commits' }
+    it { page.should have_content 
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.<br>Trailing whitespace detected.